src/services/infra/InfraService.ts

```diff
@@ -52,6 +54,9 @@ type CreateSiteParams = {
   isEmailLogin: boolean
 }
 
+const DEPRECATE_SITE_QUEUES = config.get(
+  "aws.sqs.featureFlag.shouldDeprecateSiteQueues"
```

Comment:
nit: can we call the key `featureFlags` plural

Reply:
okie 5af08260e793cc6874eb1d7728178d47648b7b54